bin/supervisord.conf

```diff
@@ -0,0 +1,25 @@
+
+[supervisord]
+nodaemon=true
+user=root
+loglevel=warn
+
+[program:create-cronjob]
```

Comment:
This is so we can run two different processes inside docker! I think this is the simpler way to make this work than to set up a separate process (in another container) that looks for indexes inside the main docker container. We don't need that kind of complexity :)